src/common/types/stripe.ts

```
@@ -33,4 +34,4 @@ export const invoiceLinkGetResponseSchema = z.array(
 );
 
 export type GetInvoiceLinksResponse = z.infer<
-  typeof invoiceLinkGetResponseSchema>;
\ No newline at end of file
+  typeof invoiceLinkGetResponseSchema>;
```

ESLint

[error] Insert `⏎`

(prettier/prettier)